### DISCUSSION TITLE
Zugriffsrechte Problem Raspi SAE

### DISCUSSION THREAD

**Discussion by NHJohnD** [camueller/SmartApplianceEnabler#404] [Q&A]
Hallo, ich bin ganz neu hier und begrüße euch alle. Da ich mich ganz frisch in diese Materie einarbeite, muss ich hier mit fragen kommen die ihr wahrscheinlich adhoc beantworten könnt.

Ich habe einen Raspberry mit SAE welches auch erreichbar ist und damit läuft. Nun möchte ich gerne meinen Shelly 3em im Sunny Portal sichtbar machen und da kommen meine Probleme. Das Sunny Portal findet das SAE nicht. In der Datei "smartapplianceenabler" steht unter dem semp.gateway.address folgendes:

#JAVA_OPTS="${JAVA_OPTS} -Dsemp.gateway.address=10.0.0.3"

Daher denke ich dass das Doppelkreuz entfernt werden muss um das zu aktivieren und die IP Adresse von meinemSHM2 eingetragen werden muss? Ist das richtig?

Falls ja, komme ich aber nicht weiter, da ich nicht die benötigten Rechte habe. Wie bekomme ich diese? Als Programm nutze WinSCP.

Wäre für eure Hilfe sehr dankbar.

**Comment by camueller** [comment]
Hast Du an Deinem Raspberry Pi WLAN aktiviert? Falls ja, kann das die Ursache sein, weil der SAE nicht weiss, ob er das WLAN-Interface oder das Ethernet-Interface als Adresse verwenden soll, unter welcher der SAE vom Sunny Home Manager angesprochen werden kann. Man kann das durch Setzen der `semp.gateway.address` lösen, wie Du ja schon herausgefunden hast. Dafür sollte man aber Linux etwas besser kennen.

Alternativ kannst Du bei der Installation des SAE auf die Aktivierung des WLANs verzichten und den Raspberry Pi via Ethernet anschliessen - dann verwendet der SAE automatisch das richtige Netzwerk-Interface.

**Comment by NHJohnD** [comment]
Hallo, ich danke dir für deine Antwort. Der Pi hängt am Netzwerkport, WLAN wird nicht genutzt. Ich bin jetzt soweit gekommen, das ich  in entsprechenden Datei die Zeile mit dem Gateway aktiviert habe und die IP von meinem Home Manager eingetragen habe. Ist das so richtig? Also die IP vom Sunny Home Manager?
